pontoon/base/tests/test_views.py

```diff
@@ -140,3 +146,42 @@ def test_no_subpage_no_stats_in_current_locale(self):
         with patch('pontoon.base.views.render', wraps=render) as mock_render:
             self.client.get(url)
             assert_true('part' not in mock_render.call_args[0][2])
+
+
+class ContributorsTests(TestCase):
+    @patch('pontoon.base.models.UserTranslationsManager.with_translation_counts')
+    @patch('pontoon.base.views.render', wraps=render)
+    def test_default_period(self, mock_render, mock_translations_manager):
+        """
+        Checks how view handles default period.
+        """
+        response = self.client.get('/contributors/')
+        assert_true(mock_render.call_args[0][2]['period'] == None)
+        assert_true(mock_translations_manager.call_args[0][0] == None)
+
+    @patch('pontoon.base.models.UserTranslationsManager.with_translation_counts')
```

Comment:
I find that using the decorator syntax for patching makes it harder to read through the test cases vs using the `with patch()` syntax within the function.

In either case, you're patching these methods for pretty much every test in this class, so you might as well patch them in `setUp` for all the methods. See https://github.com/mozilla/pontoon/blob/master/pontoon/administration/tests/test_sync_projects.py#L40 for some examples.